Formula/b/batt.rb

```diff
@@ -0,0 +1,50 @@
+class Batt < Formula
+  desc "Control and limit battery charging on Apple Silicon MacBooks"
+  homepage "https://github.com/charlie0129/batt"
+  url "https://github.com/charlie0129/batt/archive/refs/tags/v0.2.1.tar.gz"
```

Comment:
What about
```suggestion
    url "https://github.com/charlie0129/batt.git",
      tag:      "v0.2.1",
      revision: "90df4aaefb930ab05a6763c96866e8db0faf698e"
    head "https://github.com/charlie0129/batt.git"
```

Reply:
I remember remotely that Homebrew prefers a source tarball to checking out directly. Maybe related to Github complaints (look at how Homebrew migrated to using the API and not downloading the core Tap at all) but I can't find that anymore. Maybe it's fine now.

> I think it can be removed because the ipc http client is contained in the binary itself.

You meant these `get_limit.sh` and `set_limit.sh` under the hacks folder ins't actually used by the compiled binary? OK.